app/helpers/class-table.js

```diff
@@ -1,15 +1,11 @@
 import { helper } from '@ember/component/helper';
+const tableClass = {
+  '1.0 tpm': 'table-success',
+  '2.0 fault': 'table-warning',
+  '3.0 breakdown': 'table-danger',
+};
```

Comment:
Keys with dots and white space look weird... Consider to change structure:
```suggestion
const tableClasses = [
  { value: '1.0 tpm', class: 'table-success' },
  { value: '2.0 fault', class: 'table-warning' },
  { value: '3.0 breakdown', class: 'table-danger' }
];
```